.ci/run_metrics_ci.sh

```
@@ -0,0 +1,41 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+CURRENTDIR=$(dirname "$(readlink -f "$0")")
+REPORT_CMDS=("checkmetrics" "emailreport")
+
+# Verify/install report tools. These tools will
+# parse/send the results from metrics scripts execution.
+for cmd in "${REPORT_CMDS[@]}"; do
+	if ! command -v "$cmd" > /dev/null 2>&1; then
+		pushd "$CURRENTDIR/../cmd/$cmd"
+		make
+		make install
+		popd
+	fi
+done
+
+# Execute metrics scripts and report the results
+# by email.
+pushd "$CURRENTDIR/../metrics"
+	bash "run_all_metrics.sh"
```

question: I think if one of the metrics fails then the run_all_metrics.sh will fail, and this script will quit here maybe - can you confirm/nack that for me, and maybe add a comment here to make that clear.
If this script is invoked from localCI [run] action then I think we need to return the correct error code so localCI can then post the ack/nack text back to github - is that right?